Dockerfile

```diff
@@ -1,3 +1,14 @@
 FROM alpine:3.4
 
-RUN echo "dummy line for travis build testing 2"
+RUN apk update && apk add \
+    python3 \
+    git \
+    curl
+
+RUN git clone https://github.com/Netflix/conductor.git
```

Comment:
We should use specified version not master.

Reply:
fixed to `v1.8.1` (latest release)